texttospeech/snippets/long_audio_quickstart.py

```diff
@@ -12,49 +11,35 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-#
-# All Rights Reserved.
 
 from google.cloud import texttospeech
 import time
-import argparse
 
-def synthesize_long_audio(text, language_code, voice_name, parent, output_gcs_uri):
+def synthesize_long_audio(project_id, location, output_gcs_uri):
   """
   Synthesizes long input, writing the resulting audio to `output_gcs_uri`.
   
-  Example usage: synthesize_long_audio('Some input text to synthesize', 'en-US', 'en-US-Standard-A', 
-  'projects/{PROJECT_NUMBER}/locations/{LOCATION}', 'gs://{BUCKET_NAME}/{OUTPUT_FILE_NAME}.wav')
+  Example usage: synthesize_long_audio('12345', 'us-central1', 'gs://{BUCKET_NAME}/{OUTPUT_FILE_NAME}.wav')
   
   """
   # TODO(developer): Uncomment and set the following variables
-  # text = "YOUR_INPUT_TEXT"
-  # language_code = "YOUR_LANGUAGE_CODE"
-  # voice_name = "YOUR_VOICE_NAME"
-  # parent = "YOUR_PARENT_STRING"
-  # output_gcs_uri = "YOUR_OUTPUT_GCS_URI"
+  # parent = 'YOUR_PROJECT_ID'
+  # location = 'YOUR_LOCATION'
+  # output_gcs_uri = 'YOUR_OUTPUT_GCS_URI'
 
   client = texttospeech.TextToSpeechLongAudioSynthesizeClient()
 
-  input = texttospeech.SynthesisInput()
-
-  input.text = input_text
+  input = texttospeech.SynthesisInput(text="Test input. Replace this with any text you want to synthesize, up to 1 million bytes long!")
 
   audio_config = texttospeech.AudioConfig(audio_encoding=texttospeech.AudioEncoding.LINEAR16)
 
-  voice = texttospeech.VoiceSelectionParams(language_code=language_code, name=voice_name)
+  voice = texttospeech.VoiceSelectionParams(language_code="en-US", name="en-US-Standard-A")
+  
+  parent = 'projects/' + project_id + '/locations/' + location
 
   request = texttospeech.SynthesizeLongAudioRequest(parent=parent, input=input, audio_config=audio_config, voice=voice, output_gcs_uri=output_gcs_uri)
 
   operation = client.synthesize_long_audio(request=request)
-
-  attempts = 0
-  while (!operation.done and attempts < 30):
-    print("Operation is still not done. Sleeping for 5 seconds and then trying again...")
-    time.sleep(5)
-    attempts += 1
-  if attempts == 30:
-    print("It's taking a while for your operation to complete. If the input was very large, this may be okay. If not, it likely means there was a failure.")
-    return False
-  print("Your operation is complete; check your GCS bucket to find your audio file!"
-  return True
+  
+  result = operation.result(timeout=120)
+  print("\nFinished processing, check your GCS bucket to find your audio file!")
```

Comment:
What happens if the operation times out? What should the user do?

Reply:
Yes, letting the users know what to do with either comments or a try / except block would help, rather than assuming that the users would know how to debug.

Reply:
Got it. I spoke with my team, and it seems like for now, we are not supporting operation.result() for our service, which is okay per https://screenshot.googleplex.com/8FF7JWUbNxWRdm6. Given this, is there an elegant way to wait for operation.done() to be true?

Reply:
I don't think so - checking every x seconds and then sleeping could work but if there's no straightforward way to do that then the current timeout is fine.

I wanted to make sure that the users would know where to go to inspect the error. If that is assumed and well-understood by the majority of the users who would be trying out this sample then it's fine to leave as-is, otherwise it'd be good to call out the steps to check on how to see what the error was.

This is usually achieved with code with something like:
```
try:
    do_something()
except SomeError as error_message:
    print(f"Operation failed, see details: {error_message}")
```

However if that's not the case, additional info in the comments can also be helpful:
```
# If this did not print your result, it likely means that there was an error.
# Go to your console page, click logging and see what the issue was.
result = do_something()
print(f"Operation completed: {result}")
```

If this content will be included in the sample browser page (not in the snippet but in cloud.google.com page) then fine to leave as is.

Reply:
Gotcha, I actually don't think the timeout solution will work for us now either, since operation.result() is not supported. The best way for users to check the status here is probably to just check in a loop if the operation is done, and then sleep, up to a deadline. Going to implement this, and add comments to explain this to users. Let me know what you think--thanks!

Reply:
Sounds good! Setting up a retry in a loop for a few times makes sense to me.

Reply:
Cool, just added. Going to resolve if it looks good to you